tests/appverifier_tests/appverifier_tests/thread_local_test.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include <windows.h>
+
+#include <future>
+#include <thread>
+
+typedef void (*TestFunc)(const std::string&);
+
+class ThreadLocalTest : public ::testing::Test, public ::testing::WithParamInterface<std::string> {
+public:
+    void SetUp() {
+        target_device = GetParam();
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<std::string> obj) {
+        return obj.param;
+    }
+
+public:
+    std::string target_device = "";
+};
+
+TEST(LoadLibraryTest, load_free_library) {
+    std::promise<void> free_promise;
+    std::future<void> free_future = free_promise.get_future();
+    std::promise<void> thread_exit_promise;
+    std::future<void> thread_exit_future = thread_exit_promise.get_future();
+    auto shared_object = LoadLibraryA("openvino.dll");
```

Comment:
hi @xufang-lisa  I am not sure here, if "**.\test\Release\ov_appverifier_tests.exe**" and "**openvino.dll**" are not in same directory, could ov_appverifier_tests.exe open the library?, if not maybe we should tell user in the README.md or check the return value of this **LoadLibraryA** and print some message if nullptr, how do you think?

Reply:
@tiger100256-hu thank you for your comments, I have added the check of LoadLibrary's return value. In the README.md, settting openvino's environment has been mentioned.